Gemfile

```diff
@@ -105,12 +105,12 @@ gem "net-imap", require: false
 gem "net-pop", require: false
 gem "net-smtp", require: false
 
+gem "json-diff", "~> 0.4.1", require: false
```

Comment:
do we need  the `"~> 0.4.1"`?

Reply:
Just checked the gem and this release is from 2017 with zero activity on the GitHub repo since 🙈 

As it's just used in a rake task I don't see it being a problem.